addon/helpers/split-grid-filter.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+/**
+ * @module helpers
+ */
+
+/**
+ * Render the split-grid's filter template based on templateKey
+ *
+ * @function split-grid-filter
+ * @param    {object} options
+ * @returns  {string}
```

Comment:
All of the other files in this repository use Ember.String. Please change the occurrence for this property as well as for all other similar instances in this file to be consistent with the rest of the codebase. If it is desired to move away from the "Ember." syntax create an issue to be worked on separately.